docs/knowledgebase/getting-started/glossary.md

```diff
@@ -43,59 +44,85 @@ overall amount of votes wins. Notably:
+algorithm.
 
 ### Aurand
 
-A variant of Aura where authorities are shuffled randomly on each round, increasing security.
+A variant of [Aura](#aura) where [authorities](#authority) are shuffled randomly on each round,
+increasing security.
 
 ### Aurand/Ouroboros
 
-Extension of Aurand where block production involves validators competing over limited slots that
-move very quickly, where most slots are not populated, but with rare slot collisions.
+Extension of [Aurand](#aurand) where block production involves [authorities](#authority) competing
+over limited slots that move very quickly; most slots are not populated, albeit with rare slot
+collisions.
 
-### Aurand+GRANDPA
+### Aurand + GRANDPA
 
-A hybrid consensus scheme where Aurand is used for block production and short-term
-probabilistic-finality, with long-term absolute finality provided through GRANDPA.
+A hybrid [consensus](#consensus) scheme where [Aurand](#aurand) is used for block production and
+short-term probabilistic [finality](#finality), with deterministic finality provided through
+[GRANDPA](#grandpa).
 
 ---
 
 ## Blind Assignment of Blockchain Extension (BABE)
 
-Block authoring protocol similar to Aura where authorities are chosen with a verifiable random
-function (VRF) instead of a rotating list. Authorities are assigned a time slot in which they can
-select a chain and submit a new block for it. See
-https://w3f-research.readthedocs.io/en/latest/polkadot/BABE.html
+[Block authoring](#author) protocol similar to [Aura](#aura), however [authorities](#authority) win
+time slots based on a verifiable random function (VRF) as opposed to Aura's round-robin selection
+method. The winning authority can select a chain and submit a new block for it. Learn more about
+BABE by referring to its
+[official Web3 Foundation research document](https://w3f-research.readthedocs.io/en/latest/polkadot/BABE.html).
 
 ## Block
 
-A single item of a block chain (or parachain) that amalgamates a series of extrinsic data (the
-"body") together with some cryptographic information (the "header"). Blocks are arranged into a tree
-through parent pointers (implemented as cryptographic digests of the parent) and the tree is pruned
-into a list via a "fork-choice rule".
+A single element of a blockchain that [cryptographically](#cryptographic-primitives) binds a set of
+[extrinsic](#extrinsic) data (the "body") to a "[header](#header)". Blocks are arranged into a tree
+through parent pointers (implemented as a hash of the parent) and the tree is pruned into a list via
+a "fork-choice rule".
+
+## Blockchain
+
+A blockchain is a distributed network of computers that uses
+[cryptography](#cryptographic-primitives) to allow a group of participants to trustlessly come to
+[consensus](#consensus) on the [state](#state) of a system as it evolves over time. The computers
+that compose the blockchain network are called [nodes](#node).
```

Comment:
I think it's important that we define blockchain, but happy to hear what others have to say because I know that conceptually it represents a rather big leap.